pfrl/experiments/train_hrl_agent.py

```diff
@@ -48,7 +50,6 @@ def train_hrl_agent(
             obs_dict, r, done, info = env.step(action)
             # env.render()
             obs = obs_dict['observation']
-
             n_sg = agent.act_high_level(obs, fg, sg, step, t)
```

Comment:
this can be hidden in the `observe` function. Thus, my understanding is that HRL can be treated equally in train_agent.py` similar to a single agent. I need a little bit of time to come up with a good structure though.

Reply:
alright